cmd/gaia/testnets/README.md

```diff
@@ -1,151 +1,91 @@
-# Connect to a Testnet
+# Connect to the Testnet
 
-This document explains how to connect to the Testnet of a [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk/) based blockchain. It can be used to connect to the latest Testnet for the Cosmos Hub.
+This document explains how to connect to the `gaia-6001` testnet of the Cosmos Hub.
 
-NOTE: We are aware this documentation is sub-par and are actively working to
-improve both the tooling and the documentation to make this as painless as
+NOTE: We are aware this documentation is sub-par. We are working to
+improve the tooling and the documentation to make this process as painless as
 possible. In the meantime, join the
-[chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org) for technical support. Thanks very
-much for your patience :)
+[Validator Chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org)
+for technical support. Thanks very much for your patience. :)
 
-## Software Setup (Manual Installation)
-
-Follow these instructions to install the Cosmos-SDK and connect to the latest Testnet. This instructions work for both a local machine and a VM in a cloud server.
-
-If you want to run a non-validator full-node, installing the SDK on a Cloud server is a good option. However, if you are want to become a validator for the Hub's `mainnet` you should look at more complex setups, including [Sentry Node Architecture](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#how-can-validators-protect-themselves-from-denial-of-service-attacks), to protect your node from DDOS and ensure high-availability (see the [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements)). You can find more information on validators in our [website](https://cosmos.network/validators), in the [Validator FAQ](https://cosmos.network/resources/validator-faq) and in the [Validator Chat](https://riot.im/app/#/room/#cosmos_validators:matrix.org).
+## Installing Software
 
```

Comment:
You may want to consider linking to https://github.com/cosmos/cosmos-sdk/blob/rigel/spec-staking/docs/guides/sdk/install.md for how to install / or link to the ubuntu/bsd installation scripts provided in that link, since those are the easiest way for people to get started. (To reduce the amount of documentation duplication)